### PR TITLE
Bidder sees copy in "about to register" and "registration pending" modals with information about identity verification

### PR DESCRIPTION
**Pull request by yuki24**
addresses https://artsyproduct.atlassian.net/browse/AUCT-952

This PR adds a new copy for IDV sales to the registration modal.

## Todos

 * [x] Add a link to FAQ
 * [x] Add more test coverage
    * [x] Back-fill tests for `src/desktop/apps/auction/components/layout/auction_info/Registration.tsx`
    * [x] Run skipped tests in `src/desktop/apps/auction/__tests__/routes.jest.js`
 * [x] Rebase and consolidate the MP v2 client/queries once https://github.com/artsy/force/pull/5401 is merged
 * [x] Merge https://github.com/artsy/reaction/pull/3373

### User who is not verified

AUCT-952-small-from-lot

### User who is verified

AUCT-952-register-verified

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The staging version of Metaphysics has breaking changes, this might not affect your build, but could mean your queries don't work the way you expect. See below for breaking changes:</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>Please don't include .js files, we want to be using TypeScript found: <a href="https://github.com/yuki24/force/blob/AUCT-952/src/desktop/apps/auction/queries/meV2.js">meV2.js</a> and <a href="https://github.com/yuki24/force/blob/AUCT-952/src/desktop/apps/auction/queries/saleV2.js">saleV2.js</a>.</td>
    </tr>
  </tbody>
</table>


- :warning: Type 'ViewingRoomSubsectionConnection' was removed
- :warning: Type 'ViewingRoomSubsectionEdge' was removed
- :warning: Field 'isTargetSupply' was removed from object type 'Artist'
- :warning: Field 'subsectionsConnection' was removed from object type 'ViewingRoom'
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e93efe3c6624e88203d97e03d80bcd456e3b4160
</p>

File: src/desktop/apps/auction/queries/meV2.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+export function MeV2Query(sale_id, live = true) {
```
Comment (damassi) [inline src/desktop/apps/auction/queries/meV2.js:1]: `.ts`

File: src/desktop/apps/auction/routes.js [hunk 2]
```diff
@@ -25,11 +28,18 @@ export async function index(req, res, next) {
   const saleId = req.params.id
 
   try {
-    const { sale } = await metaphysics({
-      query: SaleQuery(saleId),
+    const { sale } = await MetaphysicsV2({
```
Comment (damassi) [inline src/desktop/apps/auction/routes.js:31]: is `MetaphysicsV2` a class? Why uppercase?

File: src/lib/metaphysicsV2.coffee [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  return ipAddress
+
+resolveProxies = (req) ->
+  ipAddress = resolveIPv4(req.connection.remoteAddress)
+  if req?.headers?["x-forwarded-for"]?
+    return req.headers["x-forwarded-for"] + ", " + ipAddress
+  else
+    return ipAddress
+
+metaphysics = ({ query, variables, req } = {}) ->
```
Comment (damassi) [inline src/lib/metaphysicsV2.coffee:20]: Can this be updated to reflect the `v2` function? `metaphysicsV2`

File: src/desktop/apps/auction/routes.js (hunk 2 above)
Comment (yuki24) [reply src/desktop/apps/auction/routes.js:31]: We can totally call it `metaphysicsV2 `.

File: src/lib/metaphysicsV2.coffee (hunk 3 above)
Comment (yuki24) [reply src/lib/metaphysicsV2.coffee:20]: This is file local so it's not that important to be very explicit but let's do that anyway :+1:

**Comment by yuki24** [comment]
The test and type-check failures seem to have been caused by the lack of https://github.com/artsy/reaction/pull/3373, so we will have to merge it first.

File: src/desktop/apps/auction/components/layout/auction_info/__tests__/Registration.jest.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { test } from "desktop/apps/auction/components/layout/auction_info/Regist
 
 const { Registration } = test
 
-xdescribe("auction/components/layout/auction_info/Registration.test", () => {
+describe("auction/components/layout/auction_info/Registration.test", () => {
```
Comment (yuki24) [inline src/desktop/apps/auction/components/layout/auction_info/__tests__/Registration.jest.js:6]: @bhoggard bringing back these tests as part of this PR. Luckily most of them passed with little to no fixes.

File: src/desktop/apps/auction/queries/v2/me.ts [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import { graphql } from "lib/graphql"
+
+export const meV2Query = graphql`
```
Comment (yuki24) [inline src/desktop/apps/auction/queries/v2/me.ts:3]: I tried to generate types using this trick, but is failing due to the following error:

```
$ yarn relay
...

ERROR:
Encountered 2 errors:
- Relay does not allow aliasing fields to `id`. This name is reserved for the globally unique `id` field on `Node`.

  Source: desktop/apps/auction/queries/v2/sale.ts (5:7)
  4:       _id: internalID
  5:       id: slug
           ^
  6:       associated_sale: associatedSale {
```

However, with this syntax the editor can apply syntax highlight and Prettier can also prettify the query, which is a big improvement.

**Pull request merged**